Mathlib/LinearAlgebra/Matrix/PosDef.lean

```diff
@@ -130,6 +130,14 @@ lemma PosSemidef.eigenvalues_nonneg [DecidableEq n] [DecidableEq 𝕜] {A : Matr
     (hA : Matrix.PosSemidef A) (i : n) : 0 ≤ hA.1.eigenvalues i :=
   (hA.re_dotProduct_nonneg _).trans_eq (hA.1.eigenvalues_eq _).symm
 
+lemma eigenvalues_conjTranspose_mul_self_nonneg {A : Matrix m n 𝕜} [DecidableEq n] [DecidableEq 𝕜] :
```

Comment:
I think `A` should be explicit (you can also write `(i : ...)` instead of `\forall i`.

Reply:
OK done for the explicit A. I guess the reason here is there is no way for A to be infered since there is no other argument. right?

Can you explain why (i : n) is better that \forall i? Thanks

Reply:
It's just the usual convention in mathlib, to Lean the two results are absolutely the same, see [here](https://leanprover-community.github.io/contribute/style.html#hypotheses-left-of-colon).

Reply:
Done!